docs/blog/newsletter-november-2017.rst

```diff
@@ -0,0 +1,116 @@
+.. post:: November 05, 2017
+   :tags: newsletter
+
+
+#########################################
+Write the Docs Newsletter – November 2017
+#########################################
+
+Happy November everybody! Kelly O. here, reporting live from the coffee shop that is about to cut me off because I've had three flat whites in a row. But I'm considering them celebratory beverages, because we have a jam-packed month of documentation goodness to talk about.
+
+In the next few weeks, the orangization team will be:
```

Comment:
s/orangization/organization

Unless, of course, we're all going orange this fall ...

Reply:
Oranginate, oranginate!

Reply:
ORANGINATE ORANGINATE!